qa/suites/rgw/sts/tasks/webidentity_tests/config_webidentity.yaml

```diff
@@ -3,15 +3,12 @@ tasks:
 - keycloak:
     client.0:
       keycloak_version: 11.0.0
-- s3tests:
-    client.0:
-      sts_tests: True
-      force-branch: ceph-master
-      rgw_server: client.0
-      extra_attrs: ['webidentity_test']
 overrides:
   ceph:
     conf:
       client:
               rgw sts key: abcdefghijklmnop
               rgw s3 auth use sts: true
+  s3tests:
```

Comment:
`s3tests` here is a task, so shouldn't it be nested under `tasks` and not under `overrides`?